### ISSUE TITLE
DOMAIN_EXPIRATION is negative for .co.ua domains

### ISSUE THREAD

**Issue by beliys** [TwiN/gatus#505]
### Describe the bug

It seems all *.co.ua domains are not working as expected.

### What do you see?

Results in an error where the returned DOMAIN_EXPIRATION is negative.

### What do you expect to see?

positive DOMAIN_EXPIRATION

### List the steps that must be taken to reproduce this issue

```
 - name: "xyz.co.ua"
    url: "https://xyz.co.ua"
    interval: 24h
    conditions:
      - "[DOMAIN_EXPIRATION] > 720h"
```

### Version

latest git commit

### Additional information

```
$ whois xyz.co.ua

Domain ID:1003527_COUA-DRS
Domain Name:XYZ.CO.UA
Created On:11-Dec-2020 16:47:31 UTC
Last Updated On:20-Mar-2023 13:49:31 UTC
Expiration Date:11-Dec-2024 16:47:31 UTC
Sponsoring Registrar:ТОВ "Хостінг Україна" (ukraine-mnt-cunic)
Status:ok
Registrant ID:YGMWPTEI-CUNIC
Registrant Name:<not disclosed>
Registrant Organization:<not disclosed>
Registrant Street1:<not disclosed>
Registrant Street2:<not disclosed>
Registrant Street3:<not disclosed>
Registrant City:<not disclosed>
Registrant State/Province:<not disclosed>
Registrant Postal Code:<not disclosed>
Registrant Country:<not disclosed>
Registrant Phone:<not disclosed>
Registrant Fax:<not disclosed>
Registrant Email:<not disclosed>
Admin ID:YGMWPTEI-CUNIC
Admin Name:<not disclosed>
Admin Organization:<not disclosed>
Admin Street1:<not disclosed>
Admin Street2:<not disclosed>
Admin Street3:<not disclosed>
Admin City:<not disclosed>
Admin State/Province:<not disclosed>
Admin Postal Code:<not disclosed>
Admin Country:<not disclosed>
Admin Phone:<not disclosed>
Admin Fax:<not disclosed>
Admin Email:<not disclosed>
Billing ID:YGMWPTEI-CUNIC
Billing Name:<not disclosed>
Billing Organization:<not disclosed>
Billing Street1:<not disclosed>
Billing Street2:<not disclosed>
Billing Street3:<not disclosed>
Billing City:<not disclosed>
Billing State/Province:<not disclosed>
Billing Postal Code:<not disclosed>
Billing Country:<not disclosed>
Billing Phone:<not disclosed>
Billing Fax:<not disclosed>
Billing Email:<not disclosed>
Tech ID:YGMWPTEI-CUNIC
Tech Name:<not disclosed>
Tech Organization:<not disclosed>
Tech Street1:<not disclosed>
Tech Street2:<not disclosed>
Tech Street3:<not disclosed>
Tech City:<not disclosed>
Tech State/Province:<not disclosed>
Tech Postal Code:<not disclosed>
Tech Country:<not disclosed>
Tech Phone:<not disclosed>
Tech Fax:<not disclosed>
Tech Email:<not disclosed>
Name Server:MITCH.NS.CLOUDFLARE.COM
Name Server:ARIADNE.NS.CLOUDFLARE.COM
```

**Comment by TwiN** [comment]
See comments on #504

**Comment by TwiN** [comment]
Resolved by #518
